WebSaverView.m

```diff
@@ -26,8 +26,8 @@ @implementation WebSaverView
 static NSString * const REFRESH_INTERVAL_KEY = @"RefreshInterval";
 static NSString * const REFRESH_UNITS_KEY = @"RefreshUnits";
 
-static NSString * const DEFAULT_URL = @"https://github.com/tlrobinson/WebSaver";
-static NSString * const DEFAULT_USERSCRIPT = @"document.body.style.backgroundColor = 'green';";
+static NSString * const DEFAULT_URL = @"http://webglsamples.googlecode.com/hg/aquarium/aquarium.html";
```

Comment:
I believe this URL has changed to http://webglsamples.org/aquarium/aquarium.html